src/utils/data-access.ts

```diff
@@ -0,0 +1,98 @@
+import { PromiseResult } from 'aws-sdk/lib/request';
+import { AWSError } from 'aws-sdk/lib/error';
+import DynamoDB, { DocumentClient, QueryOutput } from 'aws-sdk/clients/dynamodb';
+import { Configurations } from './configuration';
+import * as domain from '../domain';
+
+export class DataAccess {
+  private static instance: DataAccess;
+
+  private readonly tableName: string = Configurations.getInstance().dynamoTableName;
+
+  private static docClient: DocumentClient;
+
+  private constructor() {
+    console.log('config', Configurations.getInstance().dynamoTableName);
+    DataAccess.docClient = new DynamoDB.DocumentClient(Configurations.getInstance().dynamoParams);
+  }
+
+  public static getInstance(): DataAccess {
+    if (!DataAccess.instance) {
+      DataAccess.instance = new DataAccess();
+    }
+
+    return DataAccess.instance;
+  }
+
+  public getById(options: {
```

Comment:
Can this be made private and mocked when needed in the Unit Tests?

Reply:
Done